code/API_definitions/qod-api.yaml

```diff
@@ -329,18 +402,229 @@ components:
       description: |
         Ports may be specified as a list of ranges or single ports.
     QosProfile:
+      description: |
+        Data type with attributes of a QosProfile
+      type: object
+      properties:
+        name:
+          allOf:
+            - $ref: "#/components/schemas/QosProfileName"
+            - description: |
+                The name of the QoS profile.  This name is used to identify the profile when creating a session.
+                The name must be unique.
+        status:
+          allOf:
+            - $ref: "#/components/schemas/QosProfileStatusEnum"
+            - description: |
+              The status of the QoS profile.  The status is set to ACTIVE when the profile is created.
+              The status is set to INACTIVE when the profile is deleted.
+        targetMinUpstreamRate:
```

Comment:
I miss a new optional property:
```
QoSProfile:
  properties:
    description:
      description: Qualitative description of the profile
      type: string
```    
This is a demand from our Product team to leave some placeholder where to put indications or disclaimers not easily covered in other properties

Reply:
What mix / max length would you suggest for the description property?

Should we call it qosDescription or something that's not an openid keyword?

Reply:
To me `description`  is enough, as it is already under `qosProfile`, so context is clear. I don't think it is necessary to define limits explicitly, but I don't have strong opinion here, if any, a max limit in the order of 1000 would be enough.